crates/policy/src/lib.rs

```diff
@@ -85,18 +86,25 @@ impl Entrypoints {
     }
 }
 
-#[derive(Debug)]
+#[derive(Serialize, Debug)]
```

Comment:
alternatively this could have a serializeable static part plus rest as two separate properties, which feels a smidge less 'smart', but also fine with this

Reply:
ok you tricked me into writing `DataBase`